controller/test/sensor_tests/sensors_test.cpp

```diff
@@ -72,42 +91,110 @@ TEST(SensorTests, FullScaleReading) {
   // versus what the original pressure waveform was
   for (auto p : pressures) {
     SCOPED_TRACE("Pressure " + std::to_string(p.kPa()));
-    Hal.test_setAnalogPin(AnalogPin::PATIENT_PRESSURE,
-                          MPXV5004_PressureToVoltage(p));
-    Hal.test_setAnalogPin(AnalogPin::INFLOW_PRESSURE_DIFF,
-                          MPXV5004_PressureToVoltage(kPa(3.0)));
-    Hal.test_setAnalogPin(AnalogPin::OUTFLOW_PRESSURE_DIFF,
-                          MPXV5004_PressureToVoltage(kPa(1.0)));
-
-    auto readings = sensors.GetSensorReadings();
+
+    auto readings = update_readings(seconds(0.0f), p, p, p, &sensors);
```

Comment:
We're now looking at the "patient pressure" reading, so it isn't necessary in this test to bother with the inflow/outflow sensors at all.

Reply:
yes, I'll simplify this part of the test.